mitodl/mitxonline
# fix: replacing course key with course number in enroll and unenroll email

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes https://github.com/mitodl/mitxonline/issues/279

#### What's this PR do?
This PR replaces the course key with course number (i.e: the last part of course.readable_id, after the final +) from the email templates of course enrollment and unenrollment.  

#### How should this be manually tested?
- Go to mitxonline
- Enroll into a course and then unenroll
- View the enrollment and unenrollment emails to verify that course key is now replaced with course number  

#### Screenshots (if appropriate)
Course enrollment email: 
<img width="1121" alt="enrollment email" src="https://user-images.githubusercontent.com/93309234/145782612-1eca4224-5ffa-486d-a9f6-9262cdfb76f5.png">

Course enrollment email:
<img width="1133" alt="unenrollment email" src="https://user-images.githubusercontent.com/93309234/145782660-5b4a9d6e-6199-401b-9cf8-ce0d8f774e64.png">
